part2/lists.js

```diff
@@ -0,0 +1,59 @@
+document.body.style.fontFamily="Arial, sans-serif";

```

Comment:
It seems that you forget to add querySelector
```suggestion
document.querySelector('body').style.fontFamily="Arial, sans-serif";
```